### PR TITLE
ZBS297/more metadata

### PR DESCRIPTION
**Pull request by zack-erisyon**
- wip
- wip

### CODE REVIEW

File: plaster/gen/gen_main.py [hunk 1]
```diff
@@ -808,11 +808,16 @@ def main(self):
                 if report is not None:
                     utils.json_save(job_folder / f"{report_name}.ipynb", report)
 
+            dst_folder = job_folder / "reports_archive"
```
Comment (danwinkler) [inline plaster/gen/gen_main.py:811]: This is a little bit confusing as written as you set dst_folder to "reports_archive", but then later overwrite it in the loop. At first glance I thought you were putting all reports under "reports_archive". I'd either rename `dst_folder` to `reports_archive_folder`, or just do `(job_folder / "reports_archive").mkdir()`.

File: plaster/run/run.py [hunk 2]
```diff
@@ -161,39 +161,13 @@ def _tmp_dir(self):
         else:
             return local.path(local.env["HOST_ERISYON_TMP"])
 
-    def _translate_s3_references(self, task, skip_s3):
```
Comment (danwinkler) [inline plaster/run/run.py:164]: This change prevents re-running old gens. Is that okay?

**Review by danwinkler — APPROVED** [review]
see notes

File: plaster/gen/gen_main.py (hunk 1 above)
Comment (zack-erisyon) [reply plaster/gen/gen_main.py:811]: good call. due to a curt and paste issue. fixed

File: plaster/run/run.py (hunk 2 above)
Comment (zack-erisyon) [reply plaster/run/run.py:164]: I think so. We've got the gen in a better state of setting things up correctly and it feels like it is time to get s3 out of run to follow out pattern that the runner is file based.

**Pull request merged**